scripts/generateModels.mts

```diff
@@ -0,0 +1,67 @@
+import * as fsPromises from "fs/promises";
+import { generateModelsAsync, textHelper } from '@kontent-ai/model-generator';
+import * as dotenv from "dotenv";
+
+dotenv.config({ path: ".env.local" });
+
+const createMissingVarErrorMsg = (variableDescription: string) => `Missing ${variableDescription}. Please, make sure it is properly set in your .env.local file or otherwise provided as an environment variable.`
+
+type Style = "camelCase" | "pascalCase";
+
+const byNameResolver = (style: Style) => (obj: Readonly<{ name: string }>) => resolveName(obj.name, style);
+
+const resolveName = (name: string, style: Style) => replaceInvalidChars(textHelper.resolveTextWithDefaultResolver(name, style));
+
+const replaceInvalidChars = (str: string) => map(replaceIfNeeded, str)
+
+const replaceIfNeeded = (char: string, index: number) => {
+  const isValid = index === 0 ? isValidFirstChar : isValidChar;
+
+  return isValid(char) ? char : "_";
+};
+
+const firstCharRegex = /[a-zA-Z_$]/;
+const isValidFirstChar = (char: string) => firstCharRegex.test(char);
+
+const validCharRegex = /\w$/;
+const isValidChar = (char: string) => validCharRegex.test(char);
+
+const map = (mapper: (char: string, index: number) => string, str: string) =>
+  str.replaceAll(/./g, mapper);
+
+const {
+  KONTENT_MANAGEMENT_API_KEY,
+  NEXT_PUBLIC_KONTENT_ENVIRONMENT_ID,
+} = process.env;
+
+if (!KONTENT_MANAGEMENT_API_KEY) {
+  throw new Error(createMissingVarErrorMsg("management api key"));
+}
+if (!NEXT_PUBLIC_KONTENT_ENVIRONMENT_ID) {
+  throw new Error(createMissingVarErrorMsg("environment id"));
+}
+
+console.log("Deleting 'models' directory.");
+
+await fsPromises.rm(new URL("../../models", import.meta.url), { recursive: true, force: true });
+
+console.log("'models' directory deleted. Generating models.");
+
+await generateModelsAsync({
+  apiKey: KONTENT_MANAGEMENT_API_KEY,
```

Comment:
do we need this one, when sdk type is set to delivery? 
![image](https://github.com/kontent-ai-bot/kontent-ai-new-sample-app/assets/54802833/42ed182c-d4d2-4b4d-8022-24e26755427f)
from readme in delivery SDK i have a feeling that apikey is not important

Reply:
The models need to be fetched from the management API because you can't get most of the entities from the delivery API. The key is even a required property in the TypeScript type of function argument.
btw. the key is on the screenshot you sent, it's just anonymized to `yyy` 😝